test/infrastructure/docker/exp/docker/nodepool.go

```diff
@@ -282,7 +282,7 @@ func (np *NodePool) reconcileMachine(ctx context.Context, machine *docker.Machin
 		if err != nil {
 			// Requeue if there is an error, as this is likely momentary load balancer
 			// state changes during control plane provisioning.
```

Comment:
I wonder if we should maybe log the error at least to make it debuggable. (same below)

Reply:
+ to log an entry as we are doing just below for externalMachine.SetNodeProviderID